src/Squidex.Infrastructure/ComplexQuerySchemaValidationException.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System;
+
+namespace Squidex.Infrastructure
```

Comment:
I try to keep the Infrastructure free from the Squidex domain, Squidex in this case is more the company name ;)

Reply:
I see. So this class should go somewhere else?

Reply:
Yes, to the namespace with the Interfaces